server/src/models/Vote.js

```diff
@@ -0,0 +1,43 @@
+const Model = require('./Model.js')
+
+class Vote extends Model{
+    static get tableName(){
+        return 'votes'
+    }
+    
+    static get relationMappings(){
+        const Review = require('./Review.js')
+        const User = require('./User.js')
+
+        return {
+            review: {
+                relation: Model.BelongsToOneRelation,
+                modelClass: Review,
+                join: {
+                    from: 'votes.reviewId',
+                    to: 'reviews.id'
+                }
+            },
+            user: {
+                relation: Model.BelongsToOneRelation,
+                modelClass: User,
+                join: {
+                    from: 'votes.userId',
+                    to: 'users.id'
+                }
+            }
+        }
+    }
+
+    static get jsonSchema(){
+        return {
+            type: 'object',
+            required: ['voteValue'],
+            properties: {
+                voteValue: {type: ['string', 'integer'], minimum: -1, maximum: 1}
```

Comment:
This could even be created as an `enum` value so that only the options -1, 0, or 1 are allowed (not a required change, something to consider)